pkg/skuba/actions/cluster/init/init.go

```diff
@@ -115,7 +115,12 @@ func Init(initConfiguration InitConfiguration) error {
 		return errors.Errorf("cluster configuration directory %q already exists", initConfiguration.ClusterName)
 	}
 
-	scaffoldFilesToWrite := scaffoldFiles
+	scaffoldFilesToWrite := criScaffoldFiles["criconfig"]
```

Comment:
that supposes the criconfig is the same for < 18 and >= 18.
Yet criDefaultsConf file is different, it contains more things.

Could this be an "else"?

Reply:
We would like to keep this as default and drop `if` condition in CaaSP 5.0 `else` will cause branch prediction jump which is expensive in coding.

Reply:
But this crio configuration needs to be different per version, doesn't it?

Reply:
Nope, this is change regarding format configuration. We don't change it